### PR TITLE
added client side cookie for loading images

### LINKED ISSUE #2845: [Dev] Images not loading
Images are not displaying in the RTE because the cookies are being blocked on staging and production

### This task is done when…
This could include:
 - [ ] images are displaying

### PR DESCRIPTION
**Pull request by thetif**
The cookie is now being created client side. Resolved issue with creating the cookie client side so that it could be read server side. Adjusted the client side settings for creating the cookie. Changed the backend to always read from the frontend cookie.

resolves #2845 

### This pull request changes...

- expected a value cookie to be sent with image requests

### This pull request is ready to merge when...

~- [ ] Tests have been updated (and all tests are passing)~
- [ ] This code has been reviewed by someone other than the original author
~- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~
~- [ ] The change has been documented~
~- [ ] Associated OpenAPI documentation has been updated~
~- [ ] Changelog is updated as appropriate~

### This feature is done when...

~- [ ] QA has approved the experience~
~- [ ] Design has approved the experience~
~- [ ] Product has approved the experience~

### CODE REVIEW

**Review by tbolt — APPROVED** [review]
Seems fine

File: api/auth/jwtUtils.test.js [hunk 1]
```diff
@@ -69,13 +69,13 @@ tap.test('jwtUtils', async t => {
...
+        'gov.cms.eapd.api-token={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
         'example.cookie.value',
         'returns the JWT'
       ],
       [
         '/apds/1/files/12345',
-        'okta-token-storage_ACCESSTOKEN={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
+        'gov.cms.eapd.API-TOKEN={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
         'example.cookie.value',
         'returns the JWT'
```
Comment (eStar386) [inline api/auth/jwtUtils.test.js:80]: Can you add a comment explaining the JWT?

**Review by eStar386 — COMMENTED** [review]
Documenting an exemption, there is an issue with certain file sizes and the user isn't notified of the issue. This may be fixed in #2941

File: api/auth/jwtUtils.test.js (hunk 1 above)
Comment (thetif) [reply api/auth/jwtUtils.test.js:80]: I'm sorry, I just saw this. A JWT is a java web token, it's used for authentication.

**Comment by eStar386** [comment]
Test Steps to Verify : 


Precondition: Clear all cookies. 

1.Displays Consent Banner 
2. Log in to eAPD 
3. Add images 
4. Export the eAPD

**Pull request merged**